frontend/src/metabase/ui/components/typography/List/List.styled.tsx

```diff
@@ -2,15 +2,15 @@ import type { MantineThemeOverride } from "@mantine/core";
 
 export const getListOverrides = (): MantineThemeOverride["components"] => ({
   List: {
-    styles: theme => {
+    styles: () => {
       return {
         root: {
           // to revert "none" from the reset
           listStyleType: "revert",
         },
         item: {
           lineHeight: "1.5",
-          color: theme.fn.themeColor("text-dark"),
+          color: "var(--mb-color-text-primary)",
```

Comment:
This is to allow overriding bullet point texts when the card is disabled.
![image](https://github.com/user-attachments/assets/fc4c91f2-6482-45f5-b1fb-ec6197cffea2)